test/integration/registered_user_cannot_view_anothers_users_order_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class RegisteredUserCannotViewAnothersUsersOrderTest < ActionDispatch::IntegrationTest
+  include CategoryItemsSetup
+
+  test 'a registered user cannot view another users orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.